.gitignore

```diff
@@ -1,5 +1,6 @@
 target/
 logs/
+src/main/resources/static
```

Comment:
I don't think we should be ignoring this.

Reply:
Do you think we should include the files that are put into the static directory? What if they are changed?